tests/unit/nupic/encoders/date_test.py

```diff
@@ -57,69 +57,69 @@ def setUp(self):
     dayOfWeekExpected = [0,0,0,1,0,0,0]
 
     # not a weekend, so it should be "False"
-    weekendExpected = [1,1,1,0,0,0]
+    weekendExpected = [1, 0]
 
-    # time of day has radius of 4 hours and w of 5 so each bit = 240/5 min = 48min
-    # 14:55 is minute 14*60 + 55 = 895; 895/48 = bit 18.6
+    # time of day has radius of 4 hours and w of 5 so each bit = 240/5
+    # min = 48min 14:55 is minute 14*60 + 55 = 895; 895/48 = bit 18.6
     # should be 30 bits total (30 * 48 minutes = 24 hours)
-    timeOfDayExpected = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,0,0,0,0,0,0,0,0,0]
-    self._expected = numpy.array(seasonExpected + dayOfWeekExpected + weekendExpected \
-                          + timeOfDayExpected, dtype=defaultDtype)
+    timeOfDayExpected = (
+      [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,0,0,0,0,0,0,0,0,0])
```

Comment:
@oxtopus Why this and other lists in this file doesn't have space after collon while others have?